content/build/hydrobuild.md

```diff
@@ -96,14 +98,31 @@ name of the builder using the `--builder` flag.
 $ docker buildx build --builder cloud-<org>-default --tag <org>/<image> .
 ```
 
-> **Note**
->
-> Building with `--tag` loads the build result to the local image store
-> automatically when the build finishes. To build without a tag and load the
-> result, you must pass the `--load` flag.
->
-> If you use the containerd image store, you must always pass `--load` to
-> download the results, even if you build with a tag.
+Alternatively, you can set Hydrobuild as your default builder by running the
+following command:
+
+```console
+$ docker buildx use cloud-<org>-default --global
+```
+
+Changing your default builder with `docker buildx use` only changes the default
+builder for the `docker buildx build` command. The `docker build` command still
```

Comment:
I think `docker build` uses the buildx default builder set by the user, if `docker buildx install` has been done previously, which I think is the case on Docker Desktop 🤔 
Just tried and it works.
```
$ docker buildx use staging
$ docker build . -t myimage
[+] Building 0.0s (0/0)                            cloud:staging
```